myhpi/core/models.py

```diff
@@ -22,6 +23,36 @@
 from myhpi.core.widgets import AttachmentSelectWidget
 
 
+class UserProfile(models.Model):
```

Comment:
I personally don't really like the proxy models and generally prefer substituting the user model as a whole. However, this seems to be a bit tricky for existing projects (https://docs.djangoproject.com/en/4.2/topics/auth/customizing/#changing-to-a-custom-user-model-mid-project).